pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiTest2.java

```diff
@@ -102,6 +102,8 @@
     @Override
     public void setup() throws Exception {
         conf.setLoadBalancerEnabled(true);
+        conf.setTopicLevelPoliciesEnabled(true);
+        conf.setSystemTopicEnabled(true);
```

Comment:
This change is affecting all of the other tests in this file.
Do we have some side effect?
Are we going to use more and more resources for al the tests in this file?
My question is more general, sometimes we enable feature in common setup utilities, with the possibility to alter the semantics of the other tests.

Probably in this case there is no issue. Please clarify, I am still new to this suite of tests